.gitignore

```diff
@@ -1,4 +1,5 @@
-vendor/
 bin/
 .gocache/
 .go/
+.idea
```

Comment:
Using intellij? No issues here. Can we split this into a separate PR?

Reply:
Reverted by 3e0bfce